### PR TITLE
prep for mvp

### PR DESCRIPTION
**Pull request by HeroesOfOnesAndZeros**
Added raw server files and manual import files

### CODE REVIEW

File: core/cli/commands.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const build = async function (args: string[]): Promise<void> {
   console.log(args[0]);
   if (cmnd.build.test(args[0]) && cmnd.buildSsr.test(args[1])) {
     await Deno.writeTextFile(serverts, "BRIAN I CAN SEE YOU");
-
+    console.log(serverts)
```
Comment (unknownbreaker) [inline core/cli/commands.ts:49]: Can this be renamed to just `server`?

**Review by deadxears — APPROVED** [review]

**Comment by unknownbreaker** [comment]
I have comments...However, we can address them at a later time, post-MVP.

Can we renamed `serverts` to `server`?
Rememer to delete `server.tsx`.

**Review by unknownbreaker — APPROVED** [review]
I reluctantly approve this because it's technically all good.

File: core/cli/commands.ts (hunk 1 above)
Comment (unknownbreaker) [reply core/cli/commands.ts:49]: Save for later.

**Pull request merged**